### PR TITLE
AQCU-929: add rating shifts and rating shift comment table to uvhydro

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

File: R/uvhydrograph-render.R
Comment (mhines-usgs) [inline R/uvhydrograph-render.R]: These functions seem to work for ratingShifts too, should I revamp them to be named more generically? getCorrectionPositions(), parseCorrectionsLabelSpacing(), and getCorrectionArrows()

File: R/utils-parse.r
Comment (mhines-usgs) [inline R/utils-parse.r:268]: moved here from vdiagram-data

File: R/uvhydrograph-data.R
Comment (mhines-usgs) [inline R/uvhydrograph-data.R:480]: renaming this based on task description from laura to differentiate the two comment tables

**Comment by mhines-usgs** [comment]
I realize I should add tests here too, but waiting on other PR feedback.

File: R/uvhydrograph-render.R
Comment (mhines-usgs) [reply R/uvhydrograph-render.R]: maybe getEventPosition getEventLabelSpacing getEventArrows?

Comment (zmoore-usgs) [reply R/uvhydrograph-render.R]: Perhaps something like "getVerticalFlagPositions(), parseVertifcalFlagSpacing() and getVerticalFlagArrows()" ? The things they make are vertical lines with little flags on them. Not sure if that's a good name or not, if you have something better go for it! But I agree they should be renamed to something more generic now that they're being used for more than just corrections

**Review by zmoore-usgs — APPROVED** [review]
Code looks good to me, just that one set of function renames. I'd say it's good to merge once you hear back from Laura on your JIRA questions

**Comment by mhines-usgs** [comment]
Thanks! @zmoore-usgs

File: R/uvhydrograph-render.R
Comment (mhines-usgs) [reply R/uvhydrograph-render.R]: That's good enough for me, nice and generic!! Thanks.

**Pull request merged**
